lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
+
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-l;
+
+  .nx-btn-bar {
+    margin: 0;
+    text-align: right;
+  }
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  .nx-alert {
```

Comment:
This whole block should be removable with `container-vertical` on the footer

Reply:
> This whole block should be removable with `container-vertical` on the footer

The `margin: 0` can be removed but we still need to add the 24px of top margin to `nx-btn-bar` since that's custom.

Reply:
> The margin: 0 can be removed but we still need to add the 24px of top margin to nx-btn-bar since that's custom. f8cf02d

I assume this discussion will be closely tied to the analogous one from the other PR: https://github.com/sonatype/sonatype-react-shared-components/pull/124#discussion_r476682973

Reply:
> > The margin: 0 can be removed but we still need to add the 24px of top margin to nx-btn-bar since that's custom. [f8cf02d](https://github.com/sonatype/sonatype-react-shared-components/commit/f8cf02dfed188d659f551d553375f363663f0a4b)
> 
> I assume this discussion will be closely tied to the analogous one from the other PR: [#124 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/124#discussion_r476682973)

Ah, I see what you're saying now: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/9c2b33196d4e9225d698759b4cfbb6627f688d4f